### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by aultt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +210,11 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
...
 * **RestartService**: If true will restart SQL Service instance service after update. Default false.
 
+###xSQLServerDatabase
+* **Database**: (key) Database to be created or dropped
+* **Ensure**: An enumerated value that describes if Database is to be present or absent.
+* **SQLServer**: The SQL Server for the database
+* **SQLInstance**: The SQL instance for the database 
 ## Versions
 
 ### 1.4.0
```
Comment (TravisEz13) [inline README.md]: Please include a summary of changes in an unreleased version change per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request)

File: DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.schema.mof [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("xSQLServerDatabase")]
+class MSFT_xSQLServerDatabase : OMI_BaseResource
+{
+    [Key] String Database;
+    [Write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Write] String SQLServer;
```
Comment (TravisEz13) [inline DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.schema.mof:7]: You can have the same database on different servers, the server should be part of the key? 
Hmm, you want it optional, so it cant be the key?
Same with instance?

File: DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.psm1 [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    }
+
+    if($SQL)
+    {
+        # Check database exists
+        $SQLDatabase = $sql.Databases.Contains($Database)
+        $Present = $SQLDatabase
+    }
+        $returnValue = @{
+        Database = $Database
```
Comment (TravisEz13) [inline DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.psm1:73]: The return value should indicate the state of the resource.  
In this case, you should probably return ensure to indicate the presence of the resource

**Comment by TravisEz13** [comment]
It is required that you provide adequate test coverage for the code you change. [Contribution Guidelines -Adding test coverage for DSC resources](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#adding-test-coverage-for-dsc-resources)  Please work with us to come up with a plan to cover this code.

**Comment by TravisEz13** [comment]
@aultt Please resolve merge conflicts

File: DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.psm1 (hunk 3 above)
Comment (TravisEz13) [inline DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.psm1:73]: changes look good

File: DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.schema.mof (hunk 2 above)
Comment (TravisEz13) [inline DSCResources/MSFT_xSQLServerDatabase/MSFT_xSQLServerDatabase.schema.mof:7]: Discussed offline

File: README.md (hunk 1 above)
Comment (TravisEz13) [inline README.md]: changes look good here

**Comment by aultt** [comment]
@TravisEz13  Updates have been pushed please merge.  Thanks.

File: README.md [hunk 4]
```diff
@@ -246,12 +252,10 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
   - Allows to set custom or dynamic port values
 * xSQLServerRSSecureConnectionLevel
   - Additional of SQLHelper Function and error handling
-* xSQLServerRSConfig
```
Comment (KarolKaczmarek) [inline README.md:249]: why is this line and 254 removed?

File: README.md [hunk 5]
```diff
@@ -209,6 +210,11 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
...
 * **RestartService**: If true will restart SQL Service instance service after update. Default false.
 
+###xSQLServerDatabase
+* **Database**: (key) Database to be created or dropped
+* **Ensure**: An enumerated value that describes if Database is to be present or absent.
+* **SQLServer**: The SQL Server for the database
+* **SQLInstance**: The SQL instance for the database 
 ## Versions
 
 ### Unreleased
```
Comment (KarolKaczmarek) [inline README.md:220]: Can you update unreleased section?

**Comment by TravisEz13** [comment]
This was replace by PR #36.

**Pull request closed without merging**